src/Uno.Extensions.Navigation.Toolkit/Uno.Extensions.Navigation.Toolkit.UI.csproj

```diff
@@ -19,4 +19,20 @@
 	</ItemGroup>
 	
 	<Import Project="common.props" />
+	
+	<Choose>
+		<When Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='monoandroid10.0' or '$(TargetFramework)'=='monoandroid11.0'">
```

Comment:
@kazo0 this needs to include the net6-android and net6-ios tfms - I can add these later today